setup.py

```diff
@@ -48,7 +44,7 @@
         "Topic :: Terminals",
         "Topic :: Utilities",
     ],
-    description=("Python 3 library for downloading YouTube Videos."),
+    description=("Python library for downloading YouTube Videos."),
```

Comment:
I think we should keep this as python 3 personally, since we don't support python 2 any more